app/app.go

```diff
@@ -269,6 +294,9 @@ func init() {
 	}
 
 	DefaultNodeHome = filepath.Join(userHomeDir, "."+Name)
+
+	// TODO: this is a hack to make unit tests pass. Remove that after marketmap module fixes DefaultParams() call
+	marketmaptypes.DefaultMarketAuthority = "neutron1hxskfdxpp5hqgtjj6am6nkjefhfzj359x0ar3z"
```

Comment:
Please, remove the string literal and replace it with a derivation from a module name